### ISSUE TITLE
export_to exception "Unauthorized. Check username, password and tenant name/id."

### ISSUE THREAD

**Issue by therealmarv** [PyBossa/pybossa#1082]
```
Traceback (most recent call last):
File "/home/pybossa/pybossa/env/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request rv = self.dispatch_request()
File "/home/pybossa/pybossa/env/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request return self.view_functions[rule.endpoint](**req.view_args)
File "/home/pybossa/pybossa/env/local/lib/python2.7/site-packages/newrelic-2.40.0.34/newrelic/hooks/framework_flask.py", line 40, in _nr_wrapper_handler_ return wrapped(*args, **kwargs)
File "./pybossa/view/projects.py", line 1098, in export_to return {"json": respond_json, "csv": respond_csv, 'ckan': respond_ckan}[fmt](ty)
File "./pybossa/view/projects.py", line 1068, in respond_csv res = csv_exporter.response_zip(project, ty)
File "./pybossa/exporter/__init__.py", line 107, in response_zip return self.get_zip(project, ty)
File "./pybossa/exporter/__init__.py", line 88, in get_zip if not self.zip_existing(project, ty):
File "./pybossa/exporter/__init__.py", line 82, in zip_existing return uploader.file_exists(filename, self._container(project))
File "./pybossa/uploader/rackspace.py", line 135, in file_exists obj = cnt.get_object(name)
File "/home/pybossa/pybossa/env/local/lib/python2.7/site-packages/pyrax/cf_wrapper/container.py", line 113, in get_object ret = self.client.get_object(self, name)
File "/home/pybossa/pybossa/env/local/lib/python2.7/site-packages/pyrax/cf_wrapper/client.py", line 74, in _wrapped ret = fnc(self, *args, **kwargs)
File "/home/pybossa/pybossa/env/local/lib/python2.7/site-packages/pyrax/cf_wrapper/client.py", line 634, in get_object obj_info = self.connection.head_object(cname, oname)
File "/home/pybossa/pybossa/env/local/lib/python2.7/site-packages/swiftclient/client.py", line 1351, in head_object return self._retry(None, head_object, container, obj)
File "/home/pybossa/pybossa/env/local/lib/python2.7/site-packages/swiftclient/client.py", line 1253, in _retry self.url, self.token = self.get_auth()
File "/home/pybossa/pybossa/env/local/lib/python2.7/site-packages/swiftclient/client.py", line 1227, in get_auth insecure=self.insecure)
File "/home/pybossa/pybossa/env/local/lib/python2.7/site-packages/swiftclient/client.py", line 420, in get_auth auth_version=auth_version)
File "/home/pybossa/pybossa/env/local/lib/python2.7/site-packages/swiftclient/client.py", line 349, in get_auth_keystone raise ClientException(msg)
ClientException: Unauthorized. Check username, password and tenant name/id.
```

https://rpm.newrelic.com/accounts/533183/applications/2398033/traced_errors/2769107108

**Comment by alejandrodob** [comment]
I believe this can be safely close since it was also fixed by #1081.

**Comment by therealmarv** [comment]
yes. Close it for now... it did not appeared once in Newrelic and logs after upgrading. :)
